client/src/client.rs

```diff
@@ -1256,6 +1257,27 @@ pub trait RpcApi: Sized {
         }
     }
 
+    // Return the status of a UTXO set scan
+    fn scan_tx_out_set_status(&self) -> Result<Option<json::ScanTxOutStatusResult>> {
```

Comment:
⛏️ Nit: I'd remove the `set` since it's only returning the current status and not setting anything.

```suggestion
    fn scan_tx_out_status(&self) -> Result<Option<json::ScanTxOutStatusResult>> {
```